src/core_plugins/testbed/public/testbed.html

```diff
@@ -5,6 +5,10 @@
     you need to demonstate your functionality. Nothing need be preserved. -->
     <!-- CONTENT START -->
 
+    <div class="kuiControlledTable">
+      <loading-results results-name="all the things"></loading-results>
```

Comment:
In my experience, there's little benefit to tightly controlling a component's input like this, because unanticipated edge cases tend to pop up. For example, internationalization becomes more difficult because you typically need to translate entire phrases instead of substrings of a phrase. How about we change this interface to be a little more compositional:

```html
<loading-results>
  Loading all the things&hellip;
</loading-results>
```

This will mirror how I'd implement this component in React.